main.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/ayanamist/gdns-go/myip"
```

Comment:
The _correct_ way to fork and send PR should be: go get the upstream repo, than change git remote to one's own forked repo.

But anyway I can't seem to make relative import to work. I get the following message trying to `go build` the package:
> can't load package: $GOROOT/src/github.com/ayanamist/gdns-go/main.go:15:2: local import "./myip" in non-local package